lighthouse-core/scripts/legacy-javascript/summary-sizes.txt

```diff
@@ -1,170 +1,174 @@
+all-legacy-polyfills
+  175764       all-legacy-polyfills-core-js-3/main.bundle.min.js
+  125257       all-legacy-polyfills-core-js-2/main.bundle.min.js
+
```

Comment:
This was surprising. core-js@3 bytes size increased ~40% for the set of "legacy" polyfills.